designer/server/src/test/scala/pl/touk/nussknacker/test/utils/OpenAPIExamplesValidator.scala

```diff
@@ -0,0 +1,123 @@
+package pl.touk.nussknacker.test.utils
+
+import com.networknt.schema.{InputFormat, JsonSchemaFactory, SpecVersion}
+import io.circe.yaml.{parser => YamlParser}
+import io.circe.{ACursor, Json}
+import org.scalactic.anyvals.NonEmptyList
+import pl.touk.nussknacker.engine.util.Implicits.RichScalaMap
+import pl.touk.nussknacker.ui.api.InvalidExample
```

Comment:
IMO the InvalidExample case class should be defined close to the validator. Maybe we can move it to this file?